.ci/scripts/test_int_util.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+#
+# test_int_util.sh
+#
+# Library of common shell routines for PGE integration testing.
+#
+set -e
+umask 002
+
+[ -z "${WORKSPACE}" ] && WORKSPACE=$(realpath $(dirname $(realpath $0))/../..)
+[ -z "${PGE_TAG}" ] && PGE_TAG="${USER}-dev"
```

Comment:
I would like to see these four lines which are outside of any function kept within their respective scripts. As-is they become side-effects of sourcing `test_int_util.sh`, which might become confusing down the line.